static/app/views/replays/detail/replayMetaData.tsx

```diff
@@ -39,30 +64,48 @@ function ReplayMetaData({replayRecord}: Props) {
 
       <KeyMetricData>
         {replayRecord ? (
-          <React.Fragment>
+          <Fragment>
             <IconCalendar color="gray300" />
             <TimeSince date={replayRecord.startedAt} shorten />
-          </React.Fragment>
+          </Fragment>
         ) : (
           <HeaderPlaceholder />
         )}
       </KeyMetricData>
       <KeyMetricData>
         {replayRecord ? (
-          <React.Fragment>
+          <Fragment>
             <IconClock color="gray300" />
             <Duration seconds={replayRecord?.duration} abbreviation exact />
-          </React.Fragment>
+          </Fragment>
         ) : (
           <HeaderPlaceholder />
         )}
       </KeyMetricData>
       <KeyMetricData>
         {replayRecord ? (
-          <React.Fragment>
-            <IconFire color="red300" />
-            {replayRecord?.countErrors}
-          </React.Fragment>
+          <Fragment>
+            <ErrorTag
+              to={showErrorsTabHref}
+              onClick={e => {
+                e.preventDefault();
+                showErrorsTab();
```

Comment:
`show` is slightly confusing to me here, makes me think it's toggling the tab visibility or something rather than a navigation.

Reply:
Actually why have `showErrorsTab` at all instead of using `<Link>` from `react-router`?

Reply:
I could grab a `<Link>` yeah